src/entity_embedding_utils.jl

```diff
@@ -125,7 +127,8 @@ function MLJModelInterface.transform(
     fitresult,
     Xnew,
 )
-    is_embedding_enabled_type(transformer) || return Xnew
+    # if it doesn't have the property its not an entity-enabled model
+    hasproperty(transformer, :embedding_dims) || return Xnew
```

Comment:
This seems more fragile than what you had previously, ie `is_embedding_enabled_type(transformer) || return Xnew`. Is there are reason for the change?

Reply:
I just felt that what I had earlier is too much. Now I will consider bringing it back.

Reply:
Done.